drivers/filesystems/ntfs/btree.c

```diff
@@ -56,7 +56,7 @@ PrintAllVCNs(PDEVICE_EXTENSION Vcb,
 
     BytesRead = ReadAttribute(Vcb, IndexAllocationContext, 0, (PCHAR)Buffer, BufferSize);
 
-    ASSERT(BytesRead = BufferSize);
+    ASSERT(BytesRead == BufferSize);
```

Comment:
This was added by @ThFabba in 5579428b.

Reply:
Trevor Thompson actually.

Reply:
ouch, introduced in 0.4.7-dev-590-g5579428. I will put that on my list for backmerging. Good catch Serge!

Reply:
It luckily has no real world impact because the local variable BytesRead is currently not used afterwards anymore.